### ISSUE TITLE
As a user, I would like to be able to import a Dashboard

### ISSUE THREAD

**Issue by AlexJew** [architecture-building-systems/CityEnergyAnalyst#2789]
**Describe the feature you would like to have**
Hello, I have recently had all my dashboards cleared out completely. It is annoying since it requires a significant amount of time to be built. I have a backup of my _dashboard.yml_ file, and I thought it would be good to have an option for importing an existing dashboard for cases such as mine, but also for a more general purpose of reusing standard dashboards between projects.

**Describe the end purpose of this feature**
So I do not have to rebuild an entire dashboard after doing a false step or when wanting to work on another project.

**Describe the end-user of this feature**
A practicioner

**Provide Aditional information if possible**
EX. Screenshots, Links, Methods

**Comment by daren-thomas** [comment]
@AlexJew, you're totally right: This behaviour is annoying. I like the idea, let's see if we can schedule it sometime. For now, your solution to back up the `dashboard.yml` is probably a best practice. Imagine a library of dashboard templates to start with!
